worlds/witness/options.py

```diff
@@ -62,29 +72,36 @@ class ShuffleDoors(Choice):
 
 
 class DoorGroupings(Choice):
-    """If set to "none", there will be one key for every door, resulting in up to 120 keys being added to the item pool.
-    If set to "regional", all doors in the same general region will open at once with a single key,
-    reducing the amount of door items and complexity."""
+    """
+    If set to "none", there will be one key for every door, resulting in up to 120 keys being added to the item pool.
+    If set to "regional", all doors in the same general region will open at once with a single key, reducing the amount of door items and complexity.
+    """
     display_name = "Door Groupings"
     option_off = 0
     option_regional = 1
 
 
 class ShuffleBoat(DefaultOnToggle):
-    """If set, adds a "Boat" item to the item pool. Before receiving this item, you will not be able to use the boat."""
+    """
+    If set, adds a "Boat" item to the item pool. Before receiving this item, you will not be able to use the boat.
+    """
     display_name = "Shuffle Boat"
 
 
 class ShuffleDiscardedPanels(Toggle):
-    """Add Discarded Panels into the location pool.
-    Solving certain Discarded Panels may still be necessary to beat the game, even if this is off - The main example
-    of this being the alternate activation triggers in disable_non_randomized."""
+    """
+    Add Discarded Panels into the location pool.
+
+    Solving certain Discarded Panels may still be necessary to beat the game, even if this is off - The main example of this being the alternate activation triggers in disable_non_randomized.

```

Comment:
This sentence makes more sense flipped around, changes away from snake_case option name because you don't usually use that
```suggestion
    Even if this is off, solving certain Discarded Panels may still be necessary to beat the game - The main example of this being the alternate activation triggers in "Disable non randomized puzzles".
```